frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -983,6 +984,7 @@ pub mod pallet {
 
 			Self::deposit_event(Event::SolutionStored {
 				compute: ElectionCompute::Emergency,
+				origin: RawOrigin::Root,
```

Comment:
I was assuming `T::ForceOrigin::ensure_origin(origin.clone())?;` would return the `AccountId` but thats not the case. This is also though not correct -> `origin: RawOrigin::Root`. `ForceOrigin` can be different than `Root`. In Kusama its `StakingAdmin`.  

I am reverting in my opinion now to your old commit of using `Option` and setting it to `None` in cases where we don't have an AccountId. 🤦

Reply:
> I am reverting in my opinion now to your old commit of using Option and setting it to None in cases where we don't have an AccountId. 🤦

Yes please: https://github.com/paritytech/substrate/pull/13035/files#r1060103938